recipe/meta.yaml

```diff
@@ -43,12 +43,8 @@ about:
   license: MIT
   license_family: MIT
   license_file: LICENSE.md
-  summary: 'DDT (Data-Driven Tests) allows you to multiply one test case by
-   running it with different test data, and make it appear as multiple test
-   cases.'
-  description: 'DDT (Data-Driven Tests) allows you to multiply one test case by
-   running it with different test data, and make it appear as multiple test
-   cases.'
+  summary: DDT (Data-Driven Tests) allows you to multiply one test case by running it with different test data, and make it appear as multiple test cases.
```

Comment:
@beckermr did we do this? I kinda liked the text wrapping.

Reply:
Yeah this is ruamel.yaml. It respects text wrapped with pipes | but not via indentation so well.

Reply:
ugh